cypress/integration/gov-create-opp.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+
+describe('As a user authenticated via IDIR', function() {
+    beforeEach(function() {
+        // clean up db
+        cy.exec('dropdb -f digitalmarketplace')
+        // set up db
+        cy.exec('createdb digitalmarketplace')
+        cy.exec('npm run migrations:latest;')
```

Comment:
I suspect this is inelegant--I would ultimately like to just truncate the tables using [something like this](https://github.com/bcgov/cas-ciip-portal/blob/cc0920a680d1ecffe9acff41ee77d4e820f7a1a5/docs/test-helper.md), but was having trouble following

Reply:
Let's track as a tech debt issue? There's the truncate option (per example) and there's also the option of running migrations within a transaction (even faster) depending on how our backend is architected.

Reply:
https://github.com/button-inc/digital_marketplace/issues/117